### PR TITLE
Improve comments in tutorial "2. Create a contour map"

### PR DESCRIPTION
**Pull request by yvonnefroehlich**
**Description of proposed changes**

This PR aims to improve some formulations, fix some typos, and add some highlighting in the tutorial [2. Create a contour map](https://www.pygmt.org/dev/get_started/02_contour_map.html#sphx-glr-get-started-02-contour-map-py).

**Preview**: https://pygmt-dev--2654.org.readthedocs.build/en/2654/get_started/02_contour_map.html

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.
- [ ] Use underscores (not hyphens) in names of Python files and directories.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

File: examples/get_started/02_contour_map.py [hunk 1]
```diff
@@ -95,8 +95,8 @@
 # -------------
 #
 # To make it clear where the islands are located, the
-# :meth:`pygmt.Figure.coast` method can be used to color in the land. The
-# ``land`` is colored in as "lightgray", and the ``shorelines`` parameters
+# :meth:`pygmt.Figure.coast` method can be used to color in the landmasses.
```
Comment (yvonnefroehlich) [inline examples/get_started/02_contour_map.py:98]: As it is currently not consistent in the documentation, do we want to go with "landmass" or "land-mass"?

File: examples/get_started/02_contour_map.py [hunk 2]
```diff
@@ -75,35 +75,38 @@
...
 # variable named "grid") must be input again. The ``interval`` parameter sets
-# the spacing between lines (in this case, 500 meters), and the ``annotation``
-# parameter draws darker lines that are annotated with the elevation or
-# bathymetry.
+# the spacing between adjacent contour lines (in this case, 500 meters). The
+# ``annotation`` parameter annotates the contour lines corresponding to the
+# given interval (in this case, 1,000 meters) with the related values, here
+# elevation or bathymetry. By default, these contour lines are drawn thicker.
+# The appearance (thickness, color, style) of the contour lines can be adjusted
+# by specifying a desired ``pen``, which then applies to all contour lines.
```
Comment (yvonnefroehlich) [inline examples/get_started/02_contour_map.py:88]: Do we want to keep this part, or should I remove it, as the usage of `pen` is not explicitly shown in this tutorial?

Comment (michaelgrund) [reply examples/get_started/02_contour_map.py:88]: Would leave it and add that it's optional.

**Review by michaelgrund — APPROVED** [review]

File: examples/get_started/02_contour_map.py (hunk 2 above)
Comment (yvonnefroehlich) [reply examples/get_started/02_contour_map.py]: Thanks @michaelgrund. I like this solution and applied your formulation suggestion.

**Review by seisman — APPROVED** [review]

**Pull request merged**